ui/.env

```diff
@@ -1,6 +1,4 @@
- phaseControl = true
- apertureControl = true
- focusControlOnCanvas = true
- use2dCenteredPoints = true
- meshResultFormat = PNG
- WDS_SOCKET_PORT = 0
\ No newline at end of file
+REACT_APP_PHASECONTROL = true
```

Comment:
Thanks :), we should probably move these to "runtime" configuration instead of a build time. We have already moved a few of these to [server.yaml and ui.yml ](https://github.com/mxcube/mxcubeweb/tree/develop/test/HardwareObjectsMockup.xml/mxcube-web)

But this is of course a welcome fix :)

Reply:
aah, thats true! I did not even think on that :(

Reply:
No problem, one thing at the time :)